src/Share/index.tsx

```diff
@@ -0,0 +1,61 @@
+import clsx from "clsx";
+
+import { HomeIcon } from "../components/icon/home";
+import { DefaultButton } from "../components/ui/Button";
+import buttonStyles from "../components/ui/Button/index.module.css";
+import styles from "./index.module.css";
+
+export const SharePage = () => {
+	return (
+		<main className={styles.root}>
+			<p className={styles["user-name"]}>ユーザー名</p>
+
+			<div className={styles["share-container"]}>
+				<div className={styles["share-text"]}>
+					<img
+						alt="おすわりねこ"
+						src="cats/sitDown.png"
+						width={64}
+						height={65}
+					/>
+					<div>
+						<span>おつかれさまにゃ！</span>
+						<span>結果を共有してにゃ〜</span>
+					</div>
+				</div>
+
+				<a
+					className={styles["x-share-button"]}
+					href={`http://twitter.com/share?text=ランキング${"rank"}位!%20タイム${"time"}!%0a%0a&hashtags=ばーちゃるぼっくす`}
```

Comment:
```suggestion
					href={`http://twitter.com/share?text=ランキング${rank}位!%20タイム${time}!%0a%0a&hashtags=ばーちゃるぼっくす`}
```
こうじゃね？
このままだと`rank位`みたいになるくね？

Reply:
あい